tests/DependencyInjection/ConfigurationTest.php

```diff
@@ -33,7 +33,12 @@ public function testDefaultConfig(): void
         $treeBuilder = $this->configuration->getConfigTreeBuilder();
         $config = $this->processor->processConfiguration($this->configuration, [
             'docusign' => [
-                'access_token' => 'token',
+                'demo' => false,
```

Comment:
Would you mind adding a `'demo' => true,` test over the 2 concerned configuration keys?
I saw a test configuration with it but I did not see any test over the expected result.

Reply:
Test has been done in DocusignExtensionTest with demo mode